smart-contracts/test/Lock/updateKeyPricing.js

```diff
@@ -62,7 +62,7 @@ contract('Lock / updateKeyPricing', accounts => {
     assert.equal(keyPriceAfter.toFixed(), 0)
   })
 
-  describe('when the sender is not the lock owner', () => {
+  describe('when the sender does not have the LockManager role', () => {
```

Comment:
Might be worth adding a test for a lock manager which != the lock owner as well.

Reply:
Will do.